src/uu/dircolors/src/dircolors.rs

```diff
@@ -52,28 +64,78 @@ pub fn guess_syntax() -> OutputFmt {
     }
 }
 
+fn get_usage() -> String {
+    format!("{0} {1}", executable!(), SYNTAX)
+}
+
 pub fn uumain(args: impl uucore::Args) -> i32 {
     let args = args
         .collect_str(InvalidEncodingHandling::Ignore)
         .accept_any();
 
-    let matches = app!(SYNTAX, SUMMARY, LONG_HELP)
-        .optflag("b", "sh", "output Bourne shell code to set LS_COLORS")
-        .optflag(
-            "",
-            "bourne-shell",
-            "output Bourne shell code to set LS_COLORS",
+    let usage = get_usage();
+
+    /* Clap has .visible_alias, but it generates help like this
```

Comment:
I would prefer `visible_alias` since it keeps the code cleaner and it doesn't really matter if the help text is not 100% identical to GNU.

Reply:
Switched to `visible_alias`.